bundle/manifests/compliance.openshift.io_tailoredprofiles.yaml

```diff
@@ -57,13 +57,18 @@ spec:
                   description: RuleReferenceSpec specifies a rule to be selected/deselected,
                     as well as the reason why
                   properties:
+                    kind:
+                      description: Type of rule, either "Rule" or "CustomRule". "Rule"
+                        is the default if not specified.
+                      type: string
```

Comment:
I don't understand why we would support disabling a `CustomRule`. The use case for `CustomRule` is that:

1. User creates a `CustomRule` for a check that doesn't exist in the default profiles
2. User creates a `TailoredProfile` with the `CustomRule` from step 1 in the `enabledRules`

Are you able to walk me through the use case for adding a `CustomRule` to the `disabledRules`?